app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def increment_posts_counter
+    user = User.find(user_id)
+    user.update(posts_counter: user.posts_counter + 1)
+  end
```

Comment:
- [ ] Another way of doing this would be:

```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```

That way you do it with fewer lines, and fewer queries! 💪